job-service/server/src/main/java/org/verapdf/webapp/jobservice/server/controller/JobController.java

```diff
@@ -29,4 +39,9 @@ public JobDTO getJob(@PathVariable UUID jobId) throws NotFoundException {
 		return jobService.getJobById(jobId);
 	}
 
+	@GetMapping("/profiles")
+	public List<ProfileDTO> getProfiles() {
```

Comment:
this endpoint should be implemented in ProfilesController with `/api/profiles` path. The logic with available or not can be embedded into the controller (for now), ngix update required